docker/set_up_ubuntu.sh

```diff
@@ -29,20 +29,12 @@ PACKAGES_TO_INSTALL=(
     # For sentry-cli
     gawk
 
-    # For building breakpad
-    pkg-config
-    libcurl3
-    libcurl3-gnutls
-    libcurl4-gnutls-dev
-    wget
-    rsync
-
     # For nodejs and npm
     curl
 )
 
 # Install Ubuntu packages
-apt-get install -y ${PACKAGES_TO_INSTALL[@]}
+apt-get install -y "${PACKAGES_TO_INSTALL[@]}"
```

Comment:
shellcheck pointed this out to me. It needs quotes so the array contents don't get additionally broken up if there are spaces in the values.